docs/fsharp/index.md

```diff
@@ -14,45 +13,69 @@ ms.assetid: ea27fb37-dad1-4bd4-a3cc-4f5c70767ae9
-* [Language Reference](language-reference/index.md)
-* [F# Core Library Reference](https://msdn.microsoft.com/visualfsharpdocs/conceptual/fsharp-core-library-reference)
+[F# Programming Wikibook](https://en.wikibooks.org/wiki/F_Sharp_Programming) is a wikibook about learning F#. It is also a product of the F# community.
 
-## Online Reading Resources
+## Learn F# through videos
 
-* [F# for Fun and Profit Gitbook](https://swlaschin.gitbooks.io/fsharpforfunandprofit/content/) 
-* [F# Programming Wikibook](https://en.wikibooks.org/wiki/F_Sharp_Programming)
+[F# tutorial on YouTube](https://www.youtube.com/watch?v=c7eNDJN758U) is a great introduction to F# using Visual Studio, showing lots of great examples over the course of 1.5 hours.
 
-## Video Learning Resources
+[Introduction to Programming with F#](https://www.youtube.com/watch?v=Teak30_pXHk&list=PLEoMzSkcN8oNiJ67Hd7oRGgD1d4YBxYGC) is a great video series that uses Visual Studio Code as the main editor. The video series starts from nothing and ends with building a text-based RPG video game.
 
-* [Introduction to Programming with F# series on YouTube](https://www.youtube.com/watch?v=Teak30_pXHk&list=PLEoMzSkcN8oNiJ67Hd7oRGgD1d4YBxYGC)
-* [Introduction to F# series on FSharpTV](https://fsharp.tv/courses/fsharp-programming-intro/)
+[What's New in Visual Studio 2017 for F# For Developers](https://www.linkedin.com/learning/what-s-new-in-visual-studio-2017-for-f-sharp-for-developers) is a video course that shows some of the newer features for F# in Visual Studio 2017.
 
-## Further Resources
+## Other useful resources
+
+The [F# Snippets Website](http://www.fssnip.net) contains a massive set of code snippets showing how to do just about anything in F#, ranging from absolute beginner to highly advanced snippets.
+
+The [F# Software Foundation Slack](http://fsharp.org/guides/slack/) is a great place for beginners and experts alike, is highly active, and has some of world's best F# programmers available for a chat. We highly recommend joining.
+
+## The F# Software Foundation
+
+Although Microsoft is the primary developer of the F# language and its tools in Visual Studio, F# is also backed by an independent foundation, the F# Software Foundation (FSSF).
+
+The mission of the F# Software Foundation is to promote, protect, and advance the F# programming language, and to support and facilitate the growth of a diverse and international community of F# programmers.
 
-* [F# Learning Resources on fsharp.org](http://fsharp.org/learn.html)
-* [F# Snippets Website](http://www.fssnip.net)
-* [F# Software Foundation](http://fsharp.org)
\ No newline at end of file
+To learn more and get involved, check out [fsharp.org](http://fsharp.org). It's 100% free to join, and the network of F# developers in the foundation is something you don't want to miss out on!
```

Comment:
I'd remove this, and move the link to the text "..., the F# Software Foundation (FSSF)"